### ISSUE TITLE
Make it possible to change the angle with the mouse

### ISSUE THREAD

**Issue by captainbrosset** [captainbrosset/linear-gradient#8]
Dragging the gradient-line edges would be a good way to do this.
Maybe also dragging from the angle arc could work.

**Comment by mirceapiturca** [comment]
I've implemented a drag function for this. I've added a rotate handle at one end of the gradient line. Not really happy on how it moves as the offset is off at certain angles. Maybe the handle should be added at both ends and look more like the color label.
http://codepen.io/mircea/pen/MwgKaZ

If the code looks right to you I can make a pull.
Thanks

**Comment by captainbrosset** [comment]
> Maybe the handle should be added at both ends and look more like the color label.

Yes, I was about to propose: what about making the 2 end points of the gradient line draggable instead? which is the same thing.

I'll take a look at the code. Thanks for helping.

**Comment by mirceapiturca** [comment]
Sure will do that.
I do not like the position. Looking into your color label position function to see how you've added the labels.

As far as the code goes, I had to use few global variables. I would usually create a closure or abject for the drag. I guess that if this goes into DevTools it will be in its own window.
Its a fun and cool thing you made there.

**Comment by captainbrosset** [comment]
Thanks!
Yeah, eventually, when this goes into devtools, it will either be in its own window, or a commonjs-style module, so no sweat about the global variables.
